src/main/scala/de/upb/cs/swt/delphi/instanceregistry/daos/Tables.scala

```diff
@@ -0,0 +1,43 @@
+package de.upb.cs.swt.delphi.instanceregistry.daos
+import slick.jdbc.H2Profile.api._
+import slick.sql.SqlProfile.ColumnOption.NotNull
+
+class Instances(tag: Tag) extends Table[(Long, String, Long, String, String, String, String, String)](tag, "instances") {
+  def id = column[Long]("id", O.PrimaryKey, O.AutoInc) // This is the primary key column
+  def host = column[String]("host", O.Length(50), NotNull)
+  def portNumber = column[Long]("portNumber", NotNull)
+  def name = column[String]("name", O.Length(50), NotNull)
+  def componentType = column[String]("componentType", O.Length(50), NotNull)
+  def dockerId = column[String]("dockerId", O.Length(200), NotNull)
```

Comment:
As said above, please do not use ```NotNull``` here, since the dockerId in fact *can be null*.